### PR TITLE
add explicit comments for int_to_bytes and bytes_to_int

### PR DESCRIPTION
**Pull request by djrtwo**
Addresses #1395.
@mratsim is this what you meant? It seems a bit verbose as the initial header comment generally says the same thing. Can you remind me what exactly your issue was?

### CODE REVIEW

**Comment by JustinDrake** [comment]
> the initial header comment generally says the same thing

Agreed 😂

**Comment by mratsim** [comment]
I think the best would be to give an example, for example, int_to_bytes(1, little_endian) returns `0x01000000...00000000`

**Comment by dankrad** [comment]
I also feel like the naming of the functions and parameters makes their behaviour very obvious. And I am usually in favour of more verbosity.

> I think the best would be to give an example, for example, int_to_bytes(1, little_endian) returns `0x01000000...00000000`

I don't understand this, it's not a padding problem. Little endian literally means that the least significant byte comes first, so how was it confusing?

**Comment by djrtwo** [comment]
adjusted the comment to mention the endian-ness, but moved back to a 1-liner

File: specs/core/0_beacon-chain.md [hunk 1]
```diff
@@ -569,7 +569,7 @@ def int_to_bytes(n: uint64, length: uint64) -> bytes:
 ```python
 def bytes_to_int(data: bytes) -> uint64:
     """
-    Return the integer deserialization of ``data``.
+    Return the integer deserialization of ``data`` intepretted as ``ENDIANNESS``-endian.
```
Comment (protolambda) [inline specs/core/0_beacon-chain.md]:
```suggestion
    Return the integer deserialization of ``data`` intepreted as ``ENDIANNESS``-endian.
```

**Pull request merged**